ddtrace/internal/utils/deprecations.py

```diff
@@ -0,0 +1,9 @@
+class DDTraceDeprecationWarning(DeprecationWarning):
+    pass
```

Comment:
```suggestion
class DDTraceDeprecationWarning(DeprecationWarning):
    __module__ = "ddtrace"
```
?
~~~ console
❯ cat foo.py
class Foo(object):
    __module__ = "myfoo"
~~~
~~~ python
>>> from foo import Foo
>>> Foo.__module__
'myfoo'
~~~
but I haven't tested outside the REPL